clang/test/CXX/drs/cwg4xx.cpp

```diff
@@ -637,6 +641,8 @@ namespace cwg431 { // cwg431: 2.8
 
 namespace cwg432 { // cwg432: 3.0
   template<typename T> struct A {};
+  // expected-note@-1    {{template parameter is declared here}}
+  // since-cxx11-note@-2 {{template parameter is declared here}}
```

Comment:
```suggestion
  //   since-cxx11-note@-2 {{template parameter is declared here}}
```

Reply:
Hmm, why do you have this note emitted twice in C++11 and on?

Reply:
It's emitted once each by both errors in this section.

Reply:
Oh, put them closer to their respective errors then.